### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by MrBramme**
Added a little more info to the readme:
- Mermaid version: Up to you if we want to include this or not. If we do, we need to remember to update this for future versions
- How to add custom CSS support (specifically added an example for Font Awesome)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +20,29 @@ graph TD;
...
+
+## Mermaid
+
+Currently supporting Mermaid version 8.4.0.
+
+## Add custom CSS support
+
+You can use the built-in functionality to add custom CSS.
+
+For example: **Font Awesome**
```
Comment (mjbvz) [inline README.md]: Just link to existing [`markdown.styles` documentation](https://code.visualstudio.com/Docs/languages/markdown#_using-your-own-css) and show the suggested setting value itself:

```
"markdown.styles": [
    "https://use.fontawesome.com/releases/v5.7.1/css/all.css"
]
```

File: README.md [hunk 2]
```diff
@@ -17,3 +20,29 @@ graph TD;
     C-->D;
 ```
 ~~~
+
+## Mermaid
+
+Currently supporting Mermaid version 8.4.0.
```
Comment (mjbvz) [inline README.md]: supporting  -> supports

File: README.md (hunk 1 above)
Comment (MrBramme) [reply README.md]: resolved

File: README.md (hunk 2 above)
Comment (MrBramme) [reply README.md]: resolved

**Comment by mjbvz** [comment]
Thanks

**Pull request merged**